chapters/memory-layout/memory-operations/demos/array-vs-pointer/Makefile

```
@@ -0,0 +1,14 @@
+CFLAGS = -m32 -fno-PIC -Wall
+LDFLAGS = -m32 -no-pie
+
+.PHONY: all clean
+
+all: array_vs_pointer
+
+array_vs_pointer: array_vs_pointer.o
+
+array_vs_pointer.o: array_vs_pointer.c
+
+clean:
+	-rm -f *~
+	-rm -f array_vs_pointer.o array_vs_pointer
```

```suggestion
CC = gcc
CFLAGS = -Wall -g
LDFLAGS =

TARGET_EXEC = array_vs_pointer

SRCS := $(shell find $(SRC_DIRS) -name "*.c")
OBJS := $(SRCS:.c=.o)

$(info OBJS is $(OBJS))
$(info SRCS is $(SRCS))

$(TARGET_EXEC): $(OBJS)
	$(CC) $(OBJS) -o $@ $(LDFLAGS)

.o: %.c
	$(CC) $(CFLAGS) $(LDFLAGS) -c $< -o $@

.PHONY: clean

clean:
	$(RM) -r *.o $(TARGET_EXEC)
```
Maintain the same style of Makefiles throughout all exercises because these more "generic" ones are easier to extend / modify if need be.